Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -1767,19 +1767,25 @@ private static bool HandleKeywords(Material destMat, string keyword)
             if (null == t) {
                 GameObject go = new GameObject(rootGameObjectName);
                 t = go.GetComponent<Transform>();
+                
+                // Ensure any objects we create are on the same layer as the server:
```

Comment:
I think this reduces the portability of the code, as I am thinking to reuse this code in other packages.
Can we pass the layer into the function ?

Reply:
I am planning to put this code (or at least some of it) back to FilmInternalUtilities.

Reply:
Done now